code/game/turfs/open/lobotomy.dm

```diff
@@ -0,0 +1,23 @@
+/turf/open/floor/facility
```

Comment:
Same for all floors there.

Reply:
Just did what all the other plasteel tiles do and default to the plasteel.

Reply:
That's a bad idea: Removing floor will result in default tiles being dropped.